2023-03-28 17:42:46,259 INFO    MainThread:14544 [wandb_setup.py:_flush():68] Configure stats pid to 14544
2023-03-28 17:42:46,259 INFO    MainThread:14544 [wandb_setup.py:_flush():68] Loading settings from C:\Users\andre\.config\wandb\settings
2023-03-28 17:42:46,259 INFO    MainThread:14544 [wandb_setup.py:_flush():68] Loading settings from D:\10.semester\NN\neural-networks\1.zadanie\src\wandb\settings
2023-03-28 17:42:46,259 INFO    MainThread:14544 [wandb_setup.py:_flush():68] Loading settings from environment variables: {'_require_service': 'True'}
2023-03-28 17:42:46,259 INFO    MainThread:14544 [wandb_setup.py:_flush():68] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2023-03-28 17:42:46,260 INFO    MainThread:14544 [wandb_setup.py:_flush():68] Applying login settings: {'api_key': '***REDACTED***'}
2023-03-28 17:42:46,260 INFO    MainThread:14544 [wandb_init.py:_log_setup():492] Logging user logs to D:\10.semester\NN\neural-networks\1.zadanie\src\wandb\run-20230328_174246-4gz6v309\logs\debug.log
2023-03-28 17:42:46,260 INFO    MainThread:14544 [wandb_init.py:_log_setup():493] Logging internal logs to D:\10.semester\NN\neural-networks\1.zadanie\src\wandb\run-20230328_174246-4gz6v309\logs\debug-internal.log
2023-03-28 17:42:46,261 INFO    MainThread:14544 [wandb_init.py:init():532] calling init triggers
2023-03-28 17:42:46,261 INFO    MainThread:14544 [wandb_init.py:init():538] wandb.init called with sweep_config: {}
config: {}
2023-03-28 17:42:46,261 INFO    MainThread:14544 [wandb_init.py:init():554] re-initializing run, found existing run on stack: 1wdpe7pj
2023-03-28 17:42:46,264 INFO    MainThread:14544 [wandb_run.py:_finish():1854] finishing run neural-netwok-nn/cifar10/1wdpe7pj
2023-03-28 17:42:46,264 INFO    MainThread:14544 [jupyter.py:save_history():449] not saving jupyter history
2023-03-28 17:42:46,265 INFO    MainThread:14544 [jupyter.py:save_ipynb():379] not saving jupyter notebook
2023-03-28 17:42:46,265 INFO    MainThread:14544 [wandb_init.py:_jupyter_teardown():420] cleaning up jupyter logic
2023-03-28 17:42:46,265 INFO    MainThread:14544 [wandb_run.py:_atexit_cleanup():2083] got exitcode: 0
2023-03-28 17:42:46,266 INFO    MainThread:14544 [wandb_run.py:_restore():2066] restore
2023-03-28 17:42:46,266 INFO    MainThread:14544 [wandb_run.py:_restore():2072] restore done
2023-03-28 17:42:50,875 INFO    MainThread:14544 [wandb_run.py:_footer_history_summary_info():3442] rendering history
2023-03-28 17:42:50,876 INFO    MainThread:14544 [wandb_run.py:_footer_history_summary_info():3474] rendering summary
2023-03-28 17:42:50,883 INFO    MainThread:14544 [wandb_run.py:_footer_sync_info():3398] logging synced files
2023-03-28 17:42:50,891 INFO    MainThread:14544 [wandb_init.py:init():588] starting backend
2023-03-28 17:42:50,891 INFO    MainThread:14544 [wandb_init.py:init():592] setting up manager
2023-03-28 17:42:50,896 INFO    MainThread:14544 [backend.py:_multiprocessing_setup():106] multiprocessing start_methods=spawn, using: spawn
2023-03-28 17:42:50,900 INFO    MainThread:14544 [wandb_init.py:init():599] backend started and connected
2023-03-28 17:42:50,907 INFO    MainThread:14544 [wandb_run.py:_label_probe_notebook():1203] probe notebook
2023-03-28 17:42:50,907 INFO    MainThread:14544 [wandb_run.py:_label_probe_notebook():1213] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2023-03-28 17:42:50,907 INFO    MainThread:14544 [wandb_init.py:init():687] updated telemetry
2023-03-28 17:42:50,962 INFO    MainThread:14544 [wandb_init.py:init():727] communicating run to backend with 60.0 second timeout
2023-03-28 17:42:52,318 INFO    MainThread:14544 [wandb_run.py:_on_init():2134] communicating current version
2023-03-28 17:42:52,386 INFO    MainThread:14544 [wandb_run.py:_on_init():2143] got version response upgrade_message: "wandb version 0.14.0 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2023-03-28 17:42:52,386 INFO    MainThread:14544 [wandb_init.py:init():775] starting run threads in backend
2023-03-28 17:42:52,571 INFO    MainThread:14544 [wandb_run.py:_console_start():2114] atexit reg
2023-03-28 17:42:52,571 INFO    MainThread:14544 [wandb_run.py:_redirect():1969] redirect: SettingsConsole.WRAP_RAW
2023-03-28 17:42:52,571 INFO    MainThread:14544 [wandb_run.py:_redirect():2034] Wrapping output streams.
2023-03-28 17:42:52,571 INFO    MainThread:14544 [wandb_run.py:_redirect():2059] Redirects installed.
2023-03-28 17:42:52,572 INFO    MainThread:14544 [wandb_init.py:init():817] run started, returning control to user process
2023-03-28 17:42:52,573 INFO    MainThread:14544 [wandb_config.py:__setitem__():155] config set learning_rate = 0.001 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x0000021072FAC0D0>>
2023-03-28 17:42:52,573 INFO    MainThread:14544 [wandb_run.py:_config_callback():1250] config_cb learning_rate 0.001 None
2023-03-28 17:42:52,574 INFO    MainThread:14544 [wandb_config.py:__setitem__():155] config set batch_size = 32 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x0000021072FAC0D0>>
2023-03-28 17:42:52,574 INFO    MainThread:14544 [wandb_run.py:_config_callback():1250] config_cb batch_size 32 None
2023-03-28 17:42:52,574 INFO    MainThread:14544 [wandb_config.py:__setitem__():155] config set epochs = 50 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x0000021072FAC0D0>>
2023-03-28 17:42:52,574 INFO    MainThread:14544 [wandb_run.py:_config_callback():1250] config_cb epochs 50 None
2023-03-28 17:45:14,676 INFO    MainThread:14544 [wandb_setup.py:_flush():68] Configure stats pid to 14544
2023-03-28 17:45:14,676 INFO    MainThread:14544 [wandb_setup.py:_flush():68] Loading settings from C:\Users\andre\.config\wandb\settings
2023-03-28 17:45:14,676 INFO    MainThread:14544 [wandb_setup.py:_flush():68] Loading settings from D:\10.semester\NN\neural-networks\1.zadanie\src\wandb\settings
2023-03-28 17:45:14,676 INFO    MainThread:14544 [wandb_setup.py:_flush():68] Loading settings from environment variables: {'_require_service': 'True'}
2023-03-28 17:45:14,676 INFO    MainThread:14544 [wandb_setup.py:_flush():68] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2023-03-28 17:45:14,676 INFO    MainThread:14544 [wandb_setup.py:_flush():68] Applying login settings: {'api_key': '***REDACTED***'}
2023-03-28 17:45:14,677 INFO    MainThread:14544 [wandb_init.py:_log_setup():492] Logging user logs to D:\10.semester\NN\neural-networks\1.zadanie\src\wandb\run-20230328_174514-werz7wu5\logs\debug.log
2023-03-28 17:45:14,677 INFO    MainThread:14544 [wandb_init.py:_log_setup():493] Logging internal logs to D:\10.semester\NN\neural-networks\1.zadanie\src\wandb\run-20230328_174514-werz7wu5\logs\debug-internal.log
2023-03-28 17:45:14,677 INFO    MainThread:14544 [wandb_init.py:_jupyter_setup():438] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x000002106B1FD570>
2023-03-28 17:45:14,677 INFO    MainThread:14544 [wandb_init.py:init():532] calling init triggers
2023-03-28 17:45:14,678 INFO    MainThread:14544 [wandb_init.py:init():538] wandb.init called with sweep_config: {}
config: {}
2023-03-28 17:45:14,678 INFO    MainThread:14544 [wandb_init.py:init():554] re-initializing run, found existing run on stack: 4gz6v309
2023-03-28 17:45:14,679 INFO    MainThread:14544 [wandb_run.py:_finish():1854] finishing run neural-netwok-nn/cifar10/4gz6v309
2023-03-28 17:45:14,680 INFO    MainThread:14544 [wandb_run.py:_atexit_cleanup():2083] got exitcode: 0
2023-03-28 17:45:14,680 INFO    MainThread:14544 [wandb_run.py:_restore():2066] restore
2023-03-28 17:45:14,680 INFO    MainThread:14544 [wandb_run.py:_restore():2072] restore done
2023-03-28 17:45:18,414 INFO    MainThread:14544 [wandb_run.py:_footer_history_summary_info():3442] rendering history
2023-03-28 17:45:18,415 INFO    MainThread:14544 [wandb_run.py:_footer_history_summary_info():3474] rendering summary
2023-03-28 17:45:18,419 INFO    MainThread:14544 [wandb_run.py:_footer_sync_info():3398] logging synced files
